src/ActiveLogin.Authentication.BankId.AspNetCore/Models/BankIdQrCodeGenerator.cs

```diff
@@ -0,0 +1,19 @@
+using QRCoder;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Models
+{
+    public class BankIdQrCodeGenerator : IBankIdQrCodeGenerator
```

Comment:
I would call this `QRCoderBankIdQrCodeGenerator` so it's clear what implementation it uses.
Also, please write a small "intellisense documentation" with one line of what it does.

Reply:
I agree completely with the naming. Adding some documentation as well.